README.rst

```diff
@@ -1250,6 +1242,79 @@ in the settings:
        }
    }
 
+
+Add Checks and Alert settings from the device page
+--------------------------------------------------
+
+We can add checks and alert settings directly from the **device page**.
+
+To add a check, you just need to select an available `check type` as shown below:
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/device-inline-check.gif
+  :align: center
+
+
+The following example shows how to use the
+`setting <https://github.com/openwisp/openwisp-monitoring#openwisp_monitoring_metrics>`_
+to reconfigure the system for `iperf check <#iperf-1>`_ to send an alert if
+the measured **TCP bandwidth** has been less than **10 Mbit/s** for more than **2 days**.
+
+1. By default, `Iperf checks <#iperf-1>`_ do not have any alert settings defined,
+but it is easy to create one through the device page as shown below:
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/device-inline-alertsettings.gif
+  :align: center
+
+2. Now, add the following notification configuration to send an alert for **TCP bandwidth**.
+
+.. code-block:: python
+
+   # Main project settings.py
+   from django.utils.translation import gettext_lazy as _
+
+   OPENWISP_MONITORING_METRICS = {
+       'iperf': {
+           'notification': {
+               'problem': {
+                   'verbose_name': 'Iperf PROBLEM',
+                   'verb': _('Iperf bandwidth is less than normal value'),
+                   'level': 'warning',
+                   'email_subject': _(
+                       '[{site.name}] PROBLEM: {notification.target} {notification.verb}'
+                   ),
+                   'message': _(
+                       'The device [{notification.target}]({notification.target_link}) '
+                       '{notification.verb}.'
+                   ),
+               },
+               'recovery': {
+                   'verbose_name': 'Iperf RECOVERY',
+                   'verb': _('Iperf bandwidth now back to normal'),
+                   'level': 'info',
+                   'email_subject': _(
+                       '[{site.name}] RECOVERY: {notification.target} {notification.verb}'
+                   ),
+                   'message': _(
+                       'The device [{notification.target}]({notification.target_link}) '
+                       '{notification.verb}.'
+                   ),
+               },
+           },
+       },
+   }
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/alert_field_warn.png
+  :align: center
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/alert_field_info.png
+  :align: center
```

Comment:
I think it would easier to read the text in the image if the image width was lesser. In my experience, 800px wide images looks good on GitHub REAMDE. Please try cropping the image and check if it makes the image more useful.